en/mkdocs.yml

```diff
@@ -1025,19 +1020,15 @@ plugins:
               'reference/mediators/data-mapper-json-schema-specification.md' : 'https://mi.docs.wso2.com/en/4.4.0/reference/mediators/data-mapper-mediator'
               'reference/connectors/microsoft-azure-storage-connector/1.x/microsoft-azure-storage-connector-example.md' : 'https://mi.docs.wso2.com/en/4.4.0/reference/connectors/microsoft-azure-storage-connector/1.x/microsoft-azure-storage-reference/'
               'observe-and-manage/working-with-monitoring-dashboard.md' : 'https://mi.docs.wso2.com/en/4.4.0/observe-and-manage/working-with-integration-control-plane/'
-              'develop/creating-artifacts/data-services/creating-input-validators.md' : 'https://mi.docs.wso2.com/en/4.4.0/learn/learn-overview/'
```

Comment:
Don't remove these redirects since users can change the version of the URL using the dropdown. For example, if a user is on [this page for MI 4.2.0](https://mi.docs.wso2.com/en/4.2.0/develop/creating-artifacts/data-services/creating-input-validators) and switches to version 4.4.0, it results in a broken link if you remove the redirection.

Reply:
Since there is no equivalent page or an alternative, we should show the 404 page instead of redirecting. Otherwise the user won't get the idea that specific content is not applicable to 4.4.0, rather he/she will think there is an issue with our doc loading.